lib/htmlValidator.js

```diff
@@ -10,6 +10,18 @@ colors.setTheme({
 module.exports = function(app) {
   var params = app.get('params'),
       options,
+      i,
+      errorArr,
```

Comment:
Also, can we have a head for errors with all errors below it and a sub heading for warnings with all warning below it. Will provide more separation and organization